### PR TITLE
RIA-5811: AiP adds addendum - Notifications - post bundling

### PR DESCRIPTION
**Pull request by augustio**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-5811

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -452,7 +452,7 @@ bootJar {
 // UTF-8 as the default character set instead of windows-1252 which causes issues.
 // this setting only applies when running via gradle bootRun -- see web.config for the
 // java flag that configures the deployed appllications
-applicationDefaultJvmArgs = ["-Dfile.encoding=UTF-8"]
+applicationDefaultJvmArgs = ["-Dfile.encoding=UTF-8", "-Dhttps.protocols=TLSv1.2"]
```
Comment (sajjadh-v1) [inline build.gradle:455]: This seems unrelated to this work - is this done under a different ticket?

**Review by sajjadh-v1 — DISMISSED** [review]

**Review by sajjadh-v1 — APPROVED** [review]

**Review by rajeshthuraiyur — APPROVED** [review]
LGTM

**Review by sajjadh-v1 — APPROVED** [review]

**Review by sajjadh-v1 — APPROVED** [review]
